src/app/tabs/compile-tab.js

```diff
@@ -76,7 +76,21 @@ class CompileTab extends CompilerApi {
    */
 
   listenToEvents () {
-    this.compiler.event.register('compilationStarted', () => {
+    let onContentChanged = () => {
+      this.events.emit('statusChanged', {key: 'code', title: 'content changed, needs recompilation', type: 'info'})
```

Comment:
content -> Content or The content ?

Reply:
I replaced it by `the content has changed` . All the other title are lower case, let's keep it like this for now.